src/c++/uds/userLinux/uds/linux/unaligned.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * %COPYRIGHT%
```

Comment:
If this is a copy of asm/unaligned.h because the kernel moved the file, please make one or the other in our tree just #include the other, so we don't risk the two getting out of sync. I would lean towards making the new name have the real content and the old name #include it.

Reply:
The file could see code changes in the future, meaning it would have different content for different versions. The old content should not change, but we could have the new file simply include the old one.  Then the new file could be changed to incorporate new content without messing up older builds. (In practice, I think this is unlikely to matter.)

Reply:
"The file could see code changes in the future..." That's true with or without the renaming anyway.

"The old content should not change..." We need to stay compatible, but if it simplifies our driver code, why not?

Reply:
I actually thought of something else. For manifest purposes, we'll want to include one copy of this file but not both. That's easier if the two files are not linked.

I mean, I'll admit we could just include both in the manifest but that seems like an awkward thing for our userspace package to do, with one file simply being a redirect to the other.  Though I suppose we could unilaterally use the new location (or the old one) in userspace, the kernel linkage isn't that important...